src/cluster.rs

```diff
@@ -71,6 +71,25 @@ pub struct ClusterConnection {
     password: Option<String>,
     read_timeout: RefCell<Option<Duration>>,
     write_timeout: RefCell<Option<Duration>>,
+    #[cfg(feature = "tls")]
+    tls: Option<TlsMode>,
+}
+
+#[cfg(feature = "tls")]
+#[derive(Clone, Copy)]
+enum TlsMode {
+    Secure,
+    Insecure,
+}
+
+impl TlsMode {
```

Comment:
This will fail to compile when the `tls` flag is not requested:
```bash
$ REDISRS_SERVER_TYPE=tcp+tls RUST_TEST_THREADS=1 cargo test --features cluster test_parse_redis_url
   Compiling redis v0.21.3-alpha.0 (~/projects/redis-rs)
error[E0433]: failed to resolve: use of undeclared type `TlsMode`
  --> src/cluster.rs:88:13
   |
88 |             TlsMode::Insecure
   |             ^^^^^^^ use of undeclared type `TlsMode`

error[E0433]: failed to resolve: use of undeclared type `TlsMode`
  --> src/cluster.rs:90:13
   |
90 |             TlsMode::Secure
   |             ^^^^^^^ use of undeclared type `TlsMode`

error[E0412]: cannot find type `TlsMode` in this scope
  --> src/cluster.rs:85:6
   |
85 | impl TlsMode {
   |      ^^^^^^^ not found in this scope
...
```

Reply:
Ah, my bad, I'll fix that up later tonight!